Content.Server/Chat/SuicideSystem.cs

```diff
@@ -123,19 +135,52 @@ private bool EnvironmentSuicideHandler(EntityUid victim, SuicideEvent suicideEve
             return false;
         }
 
-        private void ApplyDeath(EntityUid target, SuicideKind kind)
+        private void ApplyLethalDamage(EntityUid target, DamageSpecifier? damage)
```

Comment:
Add a summary comment indicating that this function is for applying lethal damage spread out proportionally to the damage types given